### PR TITLE
Issue-1174

### PR DESCRIPTION
**Pull request by q0w**
# I have made things!

## Checklist

- [ ] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

## Related issues
#1174 

🙏 Please, if you or your company finds `dry-python` valuable, help us sustain the project by sponsoring it transparently on https://github.com/sponsors/dry-python. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks! You also need to update `wps` version 🙂

**Comment by q0w** [comment]
Mypy fails without None. Should I remove return?

File: returns/result.py [hunk 1]
```diff
@@ -372,7 +372,7 @@ def failure(self) -> _ErrorType:
 
     def _get_trace(self) -> Optional[List[FrameInfo]]:
         """Method that will be monkey patched when trace is active."""
-        return None  # noqa: WPS324
+        return  # noqa: WPS324
```
Comment (sobolevn) [inline returns/result.py:375]:
```suggestion
```

**Comment by sobolevn** [comment]
Let's try this 🙂

**Review by sobolevn — COMMENTED** [review]
Can you please also copy change from https://github.com/dry-python/returns/pull/1174 ?

**Comment by q0w** [comment]
Ok

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
thanks!

**Pull request merged**